client/driver/rkt.go

```diff
@@ -85,6 +86,13 @@ func (d *RktDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool, e
 	node.Attributes["driver.rkt.version"] = rktMatches[1]
 	node.Attributes["driver.rkt.appc.version"] = appcMatches[1]
 
+	minVersion, _ := version.NewVersion("0.14.0")
```

Comment:
I would also put this in a const/variable at the top and put a comment on why we only support 0.14.0 and above.